File: src/tls.c
/*
 * Copyright 2026 drblez AKA Ruslan Stepanenko (drblez@gmail.com)
 *
 * Lua/TSO TLS API stubs.
 *
 * Object Table:
 * | Object | Kind | Purpose |
 * |--------|------|---------|
 * | lua_tls_connect | function | Open TLS connection |
 * | lua_tls_listen | function | Optional TLS server |
 */
#include "tls.h"
#include "errors.h"

int lua_tls_connect(const char *params)
{
  (void)params;
  return LUZ_E_TLS_CONNECT;
}

int lua_tls_listen(const char *params)
{
  (void)params;
  return LUZ_E_TLS_LISTEN;
}

File: lua/tls.lua
-- Copyright 2026 drblez AKA Ruslan Stepanenko (drblez@gmail.com)
--
-- Lua/TSO TLS module stubs.
--
-- Object Table:
-- | Object | Kind | Purpose |
-- |--------|------|---------|
-- | tls.connect | function | Open TLS connection |
-- | tls.listen | function | Optional TLS server |
local tls = {}

function tls.connect(_)
  error("LUZ14001 tls.connect not implemented")
end

function tls.listen(_)
  error("LUZ14002 tls.listen not implemented")
end

return tls
